### DISCUSSION TITLE
`scheme` value

### DISCUSSION THREAD

**Discussion by aowss** [OAI/OpenAPI-Specification#2640] [Q&A]
Is the value of the `scheme` field in [Security Scheme Object](https://github.com/OAI/OpenAPI-Specification/blob/v3.1.1-dev/versions/3.1.0.md#security-scheme-object) supposed to be lower case ?

The values in the examples are all lower case but the names in the [IANA registry](https://www.iana.org/assignments/http-authschemes/http-authschemes.xhtml) are not.

**Comment by MikeRalphson** [comment]
The values used by the IANA Registry are defined as being case-insensitive:

https://github.com/OAI/OpenAPI-Specification/issues/1876#issuecomment-476818843

**Comment by handrews** [comment]
Answered, closing.
